Components/Sensors/MTK3339.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using Scarlet;
+using Scarlet.Components;
+using Scarlet.IO;
+using Scarlet.IO.BeagleBone;
+
+namespace Scarlet.Components.Sensors
+{
+    public class MTK3339 : ISensor
+    {
+        const string UPDATE_200_MSEC = "$PMTK220,200*2C\r\n";
+        const string MEAS_200_MSEC = "$PMTK300,200,0,0,0,0*2F\r\n";
+        const string GPRMC_GPGGA = "$PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0*28\r\n";
+        const string QUERY_GPGSA = "$PSRF103,02,01,00,01*27\r\n";
+
+        public float Latitude
+        {
+            get; private set;
+        }
+        public float Longitude
+        {
+            get; private set;
+        }
```

Comment:
The above two fields can be implemented in one-liners (just like the if-statements below)